spec/yaml/swiftnav/sbp/acquisition.yaml

```diff
@@ -17,8 +17,37 @@ include:
 definitions:
 
  - MSG_ACQ_RESULT:
+    id: 0x0014
+    short_desc: Satellite acquisition result
+    desc: |
+      This message describes the results from an attempted GPS signal
+      acquisition search for a satellite PRN over a code phase/carrier
+      frequency range. It contains the parameters of the point in the
+      acquisition search space with the best signal-to-noise (SNR)
+      ratio.
+    fields:
+        - snr:
+            type: float
+            desc: |
+              SNR of best point. Currently dimensonless, but will have
```

Comment:
Yes, will require changing the threshold. I've made an issue https://github.com/swift-nav/piksi_bringup/issues/7